tests/warn/i23693.scala

```diff
@@ -0,0 +1,19 @@
+//> using options -Wtostring-interpolated
+
+// verify warning messages and runtime result
```

Comment:
I guess "and runtime result" is inaccurate here, isn't it?

Reply:
Because I had to split the test between warn and run, for the rest of time, or until the test rig is improved, these tests must remain linked like "Ren and Ben" in Star Wars.